University-advisor-web/Models/ImageModel.cs

```diff
@@ -0,0 +1,40 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace University_advisor_web.Models
+{
+    public class ImageModel
+    {
+        public int ImageId { get; set; }
+        public string ImageAddress { get; set; }
+        public string Title { get; set; }
+        public string AuthorCredit { get; set; }
+        public ImageModel(){ }
+        public ImageModel(int iconId, string imageAddress, string title, string authorCredit = "") {
+            ImageId = iconId;
+            ImageAddress = imageAddress;
+            Title = title;
+            AuthorCredit = authorCredit;
+        }
+        public List<ImageModel> GetAllIcons() 
+        {
+            var sqlSelectIcons = SqlDriver.Fetch("SELECT * FROM icons");
+            var iconsList = new List<ImageModel>();
+            foreach (var icon in sqlSelectIcons)
+            {
+                ImageId = Convert.ToInt32(icon["iconId"].ToString());
```

Comment:
Convert is going to return 0 if provided value is null. I would suggest using TryParse extension method that is available in Tools directory. You can call TryParse and provide value that needs to be returned if parse fails because here 0 will be returned each time when convert fails. However, if there is no need to assign unique image id, you may leave it as it is.

Reply:
By idea, the imageId will never be null because it is set to 'not null' and 'auto increment' in the database. Correct imageId is required to display correct icon.

Reply:
Then you are right, no reason to worry about null values.